fortuna/prob_model/posterior/normalizing_flow/normalizing_flow_trainer.py

```diff
@@ -232,3 +209,14 @@ def validation_step(
                 **{f"val_{m}": v for m, v in val_metrics.items()},
             }
         return dict(val_loss=loss)
+
+    def _get_params_from_rav(self, _rav, unravel) -> Params:
+        if self._which_params is None:
+            return unravel(_rav)
+        return FrozenDict(
+            nested_set(
```

Comment:
[nit] can you specify the name of the arguments when you have a function call with multiple args? It greatly improved readability and is safer.